t/unit/tasks/test_result.py

```diff
@@ -367,6 +368,28 @@ def test_ready(self):
 
         assert not self.app.AsyncResult(uuid()).ready()
 
+    def test_get_request_meta(self):
+
+        x = self.app.AsyncResult('1')
+        request = Context(
+            task='foo',
+            children=None,
+            args=['one', 'two'],
+            kwargs={'kwarg1': 'three'},
+            hostname="foo",
+            retries=1,
+            delivery_info={'routing_key': 'celery'}
+        )
+        x.backend.store_result(task_id="1", result='foo', state=states.SUCCESS,
+                               traceback=None, request=request)
+        assert x.name
```

Comment:
I would suggest to be more explicit on these assertions:

```
assert x.args == ['one', 'two']
```

What do you think?

Reply:
done!